### PR TITLE
add a note in the README about fonts

### PR DESCRIPTION
**Pull request by ryanlerch**

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
:+1:  for me

**Comment by pypingou** [comment]
I have installed the fonts, but I'm not seeing a difference in the browser

**Comment by pypingou** [comment]
Ok, after restarting firefox the font seems to be different.

Could we mention it here as well?

File: README.rst [hunk 1]
```diff
@@ -45,6 +45,17 @@ elections server::
 Once that is running, simply go to http://localhost:5002/ in your browser on
 your host to see your running elections test instance.
 
+A note about Fonts
```
Comment (bowlofeggs) [inline README.rst]: Nitpick: maybe lowercase the F in Fonts so it's consistent with the style of the rest of this heading?

File: README.rst [hunk 2]
```diff
@@ -45,6 +45,17 @@ elections server::
...
+~~~~~~~~~~~~~~~~~~
+Fedora elections uses webfonts pulled down from Fedora infrastructure that might
+not work when hacking locally due to CORS resctrictions. Installing the fonts on
+the system that you are looking at the elections website on will be the same as
```
Comment (bowlofeggs) [inline README.rst]: Nitpick: maybe "the system that you are developing the elections website on"?

File: README.rst [hunk 3]
```diff
@@ -45,6 +45,17 @@ elections server::
...
+when it gets deployed to prod. Install the required fonts on Fedora with:
+
+    sudo dnf install open-sans-fonts
+
+
+
```
Comment (bowlofeggs) [inline README.rst]: Nitpick: Maybe two spaces here instead of three. Even though this isn't Python code, I love PEP-8 ☺

File: README.rst (hunk 1 above)
Comment (ryanlerch) [inline README.rst]: fixed!

File: README.rst (hunk 2 above)
Comment (ryanlerch) [inline README.rst]: I specifically chose this language, as to make the distiction between the machine you are developing on (which could be the vagrant-based guest), and the host, which is the most likely that you are browsing to the website. For the fonts to show, you need to just have them installed on the machine that you are browsing to the test site on, not neccecarily the server that is hosting the elections site.

File: README.rst (hunk 3 above)
Comment (ryanlerch) [inline README.rst]: fixed!

**Pull request merged**